Client/Frontend/Home/FxHomeJumpBackInView.swift

```diff
@@ -83,13 +83,21 @@ class FxHomeJumpBackInCollectionCell: UICollectionViewCell {
 extension FxHomeJumpBackInCollectionCell: UICollectionViewDataSource {
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
         viewModel.updateDataAnd(layoutVariables)
-        return viewModel.jumpableTabs.count
+        return viewModel.jumpList.itemsToDisplay
     }
 
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: JumpBackInCell.cellIdentifier, for: indexPath) as! JumpBackInCell
 
-        if let item = viewModel.jumpableTabs[safe: indexPath.row] {
+        if indexPath.row == (viewModel.jumpList.itemsToDisplay - 1),
+           let group = viewModel.jumpList.group {
+
+            cell.itemTitle.text = group.searchTerm.localizedCapitalized
+            cell.itemDetails.text = String(format: .FirefoxHomeJumpBackInSectionGroupSiteCount, group.groupedItems.count)
+            cell.heroImage.image = UIImage(imageLiteralResourceName: "recently_closed").withRenderingMode(.alwaysTemplate)
+
+        } else {
+            let item = viewModel.jumpList.tabs[indexPath.row]
```

Comment:
nit: maybe you can add `[safe: indexPath.row]` here?

Reply:
I'm happy to make this change, and all for it, but I don't understand what it brings to the table? If the cell count is based on the number of items in the jumpList, then `index.row` can never be greater than the number of items. So, then, is the `safe` subscript not redundant and without meaningful purpose?

Reply:
Yeah that's true - in that case we can skip these.